mne/io/eyelink/eyelink.py

```diff
@@ -267,9 +267,13 @@ def __init__(
         eye_ch_data = self.dataframes["samples"][ch_names]
         eye_ch_data = eye_ch_data.to_numpy().T
         super(RawEyelink, self).__init__(
-            info, preload=eye_ch_data, filenames=[self.fname], verbose=verbose
+            info,
+            preload=eye_ch_data,
+            filenames=[self.fname],
+            verbose=verbose,
+            raw_extras=self._raw_extras,
```

Comment:
```suggestion
            raw_extras=[raw_extras],
```
The super method will set the `self._raw_extras` attribute. See for example how other readers do it like 

https://github.com/mne-tools/mne-python/blob/1ee327014b9389962b70331f41b7de979e01140b/mne/io/brainvision/brainvision.py#L111-L119